### PR TITLE
Make the centos image as the default

### PR DESCRIPTION
**Pull request by yaroslavm**
## Related Issue
issue #101 

## Types of changes

- [x] Updated docs / Refactor code / Added a tests case / Automation (non-breaking change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,4 +92,4 @@ Image name         | Description | Layers | quay.io | docker.io
 `:latest`          | represents the master branch | [![Layers info](https://images.microbadger.com/badges/image/radanalyticsio/spark-operator:latest.svg)](https://microbadger.com/images/radanalyticsio/spark-operator:latest) |  | 
 `:x.y.z`           | one particular released version | [![Layers info](https://images.microbadger.com/badges/image/radanalyticsio/spark-operator:0.1.5.svg)](https://microbadger.com/images/radanalyticsio/spark-operator:0.1.5) |  | 
 
-For each variant there is also available an image with `-slim` suffix based on Alpine for instance [![Layers info](https://images.microbadger.com/badges/image/radanalyticsio/spark-operator:latest-released-slim.svg)](https://microbadger.com/images/radanalyticsio/spark-operator:latest-released-slim)
+For each variant there is also available an image with `-alpine` suffix based on Alpine for instance [![Layers info](https://images.microbadger.com/badges/image/radanalyticsio/spark-operator:latest-released-slim.svg)](https://microbadger.com/images/radanalyticsio/spark-operator:latest-released-slim)
```
Comment (Jiri-Kremser) [inline README.md]: `latest-released-slim` -> `latest-released-alpine`

**Comment by Jiri-Kremser** [comment]
good news, the main test suites already pass with the centos image (https://travis-ci.org/radanalyticsio/spark-operator/jobs/436640331#L2581)

don't worry about the `Deploy` failing in the travis CI, it shouldn't be run for PRs actually, it's a bug

**Comment by yaroslavm** [comment]
@Jiri-Kremser ok, great. In addition I just fixed two links to svg images in Readme by last commit.

**Comment by yaroslavm** [comment]
@Jiri-Kremser sure, looks better now. What do you think?

**Comment by yaroslavm** [comment]
@Jiri-Kremser thank you too! I'm glad to help!

**Pull request merged**